go/vt/vtgate/executor_select_test.go

```diff
@@ -160,8 +160,9 @@ func TestSystemVariablesMySQLBelow80(t *testing.T) {
 	executor.normalize = true
 
 	sqlparser.MySQLVersion = "57000"
+	*setVarEnabled = true
```

Comment:
shouldn't we add a defer to set `setVarEnabled` to false?

Reply:
next PR I can do it as it is not important.